client/webserver/site/src/js/markets.ts

```diff
@@ -226,7 +226,7 @@ export default class MarketsPage extends BasePage {
     // TODO: Use dexsettings page?
     const registerBttn = Doc.tmplElement(page.notRegistered, 'registerBttn')
     bind(registerBttn, 'click', () => {
-      app().loadPage('register', { host: this.market.dex.host })
+      if (this.market?.dex) app().loadPage('register', { host: this.market.dex.host })
```

Comment:
Unnecessary. The only time `page.notRegistered` is shown is in `setRegistrationStatusVisibility` which already checks for `this.market` and `this.market.dex`.